src/xdocs/config_coding.xml

```diff
@@ -2669,7 +2669,7 @@ class TestMethodCall {
       </subsection>
     </section>
 
-    <section name="MissingCtor">
+      <section name="MissingCtor">
```

Comment:
Your IDE did something you didn't expect. Please, redo this indentation.

Reply:
corrected.